ui/pages/desktop-pairing/index.scss

```diff
@@ -3,126 +3,39 @@
   flex-flow: column;
   align-items: center;
   padding: 0 30px 0;
+  max-width: 640px;
 
   &__countdown-timer {
-    background: #f2f3f4;
-    border-radius: 15.5px;
-    padding: 7px 0 7px 0;
-    margin: 0 32px 0 32px;
+    background: var(--color-background-default);
+    border-radius: 16px;
+    padding: 8px 0;
     font-size: 12px;
+    width: 240px;
   }
 
   &__countdown-timer-seconds {
     color: var(--color-primary-default);
   }
 
-  &__icon {
-    padding-top: 24px;
-  }
-
-  &__title {
-    font-style: normal;
-    font-weight: 700;
-    font-size: 24px;
-    line-height: 140.62%;
-    text-align: center;
-    color: #24292e;
-    padding-top: 24px;
-  }
-
-  &__subtitle,
-  &__description {
-    font-style: normal;
-    font-weight: 400;
-    font-size: 14px;
-    line-height: 140.62%;
-    text-align: center;
-    color: #000;
-    padding-top: 8px;
-    margin: 0 56px;
-  }
-
-  &__description {
-    margin: 18px 0;
-  }
-
   &__otp {
     font-style: normal;
     font-weight: 500;
     font-size: 48px;
+    line-height: 48px;
     letter-spacing: 10px;
```

Comment:
Can these styles be removed for a TextVariant and just leave the letter-spacing to reduce css?
`varian={TextVariant.displayMd}` = 48px font-size on large screens

Reply:
Good point, I will also leave `font-weight` here, because it feels way more readable, what do you think? 
- with `font-weight: 500;`
<img width="358" alt="Screenshot 2023-03-21 at 11 12 36" src="https://user-images.githubusercontent.com/7644512/226576363-ee5e39ef-cd78-467c-962d-ea1951ed337f.png">

- without `font-weight: 500;`
<img width="379" alt="Screenshot 2023-03-21 at 11 12 49" src="https://user-images.githubusercontent.com/7644512/226576423-c55b6a93-c276-4eae-bb0c-50e226a7dc0f.png">

Reply:
I would leave it without as that's how it's defined and intended to appear on small view ports. cc @Akatori-Design

Reply:
Sure, that's removed too.